config.ru

```diff
@@ -0,0 +1,65 @@
+require 'pg'
+
+def convert_keys(table)
+  table.map { |row| row.transform_keys { |key| key.to_sym rescue key } }
+end
+
+conn = PG::Connection.open(ENV["DATABASE_URL"])
+table = convert_keys(conn.exec('SELECT * from accounts'))
+
+app = -> (env) do
+  status = "<h1> Status account</h1>
+    <table>
+        <tr>
+          <th>Type</th>
+          <th>Name</th>
+          <th>Currency</th>
+          <th>Amount</th>
+        </tr>
+        <tr>
+          <td>#{table[0][:type]}</td>
```

Comment:
So, if a new account added, we should change the code? This won't work. Use `map` iterator to generate list of `tr`'s and then join them into a string.